lua/astrocommunity/pack/nix/init.lua

```diff
@@ -23,16 +23,24 @@ return {
   {
     "AstroNvim/astrolsp",
     optional = true,
-    opts = function(_, opts) opts.servers = require("astrocore").list_insert_unique(opts.servers, { "nixd" }) end,
-  },
-  {
-    "stevearc/conform.nvim",
-    optional = true,
-    opts = {
-      formatters_by_ft = {
-        nix = { "alejandra" },
-      },
-    },
```

Comment:
Unsure about how we should do this. I removed conform since that made the most sense, but I am open to adding ternary logic for astrolsp to disable formatting if conform is defined in the config